packages/core/src/runtime-main.ts

```diff
@@ -1,3 +1,5 @@
+import "./polyfills/report-error-polyfill.js";
```

Comment:
No need for it in web workers?

Reply:
no, I verified that this entrypoint is run for any type of environment, including webworker